### ISSUE TITLE
GUI accessibility: Modal dialogs contain aria-hidden

### ISSUE THREAD

**Issue by leonardder** [privacyidea/privacyidea#2224]
### Top-level intent
As a blind user, I'm trying to operate the PrivacyIdea GUI with the {NVDA screen Reader](https://www.nvaccess.org/). Currently, right after login, I get the no token modal dialog. However, this dialog isn't exposed to the screen reading software as [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute) is set to true. This blocks any screen reading software from reading the dialog, as aria-hidden tells the browser/screen reader that the dialog should be treated as hidden by screen reading software.

### Expected outcome
aria-hidden should be removed from all modal dialogs, unles they are really hidden.
For additional details on how to create accessible dialogs, see https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html 

### Actual outcome
Modal dialogs are inaccessible by screen reading software as long as aria-hidden is true.

### Configuration

* **privacyIDEA version**: 3.3.3
* **Installation method**: from Ubuntu packages

**Comment by melanger** [comment]
https://github.com/privacyidea/privacyidea/blob/7199be70bbe1ebee908518a8b529ac3637a86ca1/privacyidea/static/components/user/views/dialog.ask_user_delete.html#L6
https://github.com/privacyidea/privacyidea/blob/7199be70bbe1ebee908518a8b529ac3637a86ca1/privacyidea/static/components/dialogs/views/dialog.autocreate_realm.html#L6
https://github.com/privacyidea/privacyidea/blob/7199be70bbe1ebee908518a8b529ac3637a86ca1/privacyidea/static/components/token/views/dialog.ask_token_delete.html#L6
https://github.com/privacyidea/privacyidea/blob/7199be70bbe1ebee908518a8b529ac3637a86ca1/privacyidea/static/components/dialogs/views/dialog.lock.html#L6
https://github.com/privacyidea/privacyidea/blob/7199be70bbe1ebee908518a8b529ac3637a86ca1/privacyidea/static/components/dialogs/views/dialog.no.token.html#L10

I guess the fix would be to set `aria-hidden=false` by JavaScript when the dialog is shown, but that is probably not done now. The only place in JavaScript where `aria-hidden` is set to false is here, when something is collapsed, but that probably does not apply to dialogs:
https://github.com/privacyidea/privacyidea/blob/7199be70bbe1ebee908518a8b529ac3637a86ca1/privacyidea/static/contrib/js/ui-bootstrap-tpls.js#L42
https://github.com/privacyidea/privacyidea/blob/7199be70bbe1ebee908518a8b529ac3637a86ca1/privacyidea/static/contrib/js/ui-bootstrap-tpls.js#L64
